app/styles/partials/utils.scss

```diff
@@ -7,3 +7,6 @@
 .scrollable-modal {
   max-height: 50vh;
 }
+.ui.basic.green.label.l-margin {
```

Comment:
@CosmicCoder96 No its not working with that. I have to include all that to make it work